src/routes/jira/jira-delete.test.ts

```diff
@@ -60,7 +60,7 @@ describe("DELETE /jira/configuration", () => {
 			}
 		};
 
-		const res = { sendStatus: jest.fn(), locals: { installation, jiraHost } };
+		const res = { sendStatus: jest.fn(), locals: { installation, jiraHost, gitHubAppConfig } };
```

Comment:
I don't think this is needed at all. Its not used anywhere in `JiraDelete`

Reply:
True true. It was when I initially opened this pr but have since changed the implementation. I've removed it but will leave the changes to the test utils models and setup so those are already updated when we need to use it.